previous_code/gflownetenv.py

```diff
@@ -307,7 +315,7 @@ def true_density(self):
           - states
         """
         return (None, None, None)
-
+    #To be rather put in the Buffer ? 
```

Comment:
But isn't it environment specific? the train set of grid is really different from that of the enviroment

Reply:
Okay, I confused make_train_set for the training of the gflownet indepently of the activelearning pipeline and the functionnality of the Buffer to define a train set from which to sample offline training samples for the gflownet. Have to check this out.